Library/Homebrew/test/test_tap.rb

```diff
@@ -154,7 +154,7 @@ def test_git_variant
 
     assert_equal "e1893a6bd191ba895c71b652ff8376a6114c7fa7", @tap.git_head
     assert_equal "e189", @tap.git_short_head
-    assert_match %r{years ago}, @tap.git_last_commit
+    assert_match /years ago/, @tap.git_last_commit
```

Comment:
Could this just be `"years ago"` & avoid `//` entirely given we're not actually using a regex here or am I missing something?